apps/launcher/main.cpp

```diff
@@ -12,13 +13,29 @@
 #define MAC_OS_X_VERSION_MIN_REQUIRED __ENVIRONMENT_MAC_OS_X_VERSION_MIN_REQUIRED__
 #endif // MAC_OS_X_VERSION_MIN_REQUIRED
 
+#ifdef _WIN32
+#include <Windows.h>
+#endif
```

Comment:
Can you get away with `WIN32_LEAN_AND_MEAN` here? It's supposed to shave a significant amount off compile times.

In fact, are you 100% sure you need to do this at all? Qt should have cross-platform DPI scaling stuff that doesn't need you to include anything OS-specific.

Reply:
Good point for `#define WIN32_LEAN_AND_MEAN`. I forgot. 
Yes, it's the only true solution for support High DPI on Windows if we have multiple screens. To my knowledge. 

Because ` QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);` don't resize correctly some inputs (like checkbox) and the Windows window border stay very small. And QMessageBox are not resized correctly. It's working only if we have only one screen. Or multiple High DPI screens.

With this code. No problem. Other solution (but similar) is to add a parameter in the manifest of the Windows Application. It's do the same. But directly implement the code seems to me more comprehensive (because currently applications don't have it).